common/arg.cpp

```diff
@@ -3423,5 +3423,36 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
         }
     ).set_examples({LLAMA_EXAMPLE_SERVER}));
 
+    // diffusion parameters
+    add_opt(common_arg({ "--diffusion-steps" }, "N",
+                       string_format("number of diffusion steps (default: %d)", params.diffusion.steps),
+                       [](common_params & params, int value) { params.diffusion.steps = value; })
+                .set_examples({ LLAMA_EXAMPLE_DIFFUSION }));
```

Comment:
Fix the pattern here:

```suggestion
    // diffusion parameters
    add_opt(common_arg(
        { "--diffusion-steps" }, "N",
        string_format("number of diffusion steps (default: %d)", params.diffusion.steps),
        [](common_params & params, int value) { params.diffusion.steps = value; }
    ).set_examples({ LLAMA_EXAMPLE_DIFFUSION }));
```